packages/oscal-react-library/src/components/oscal-utils/OSCALLinkUtils.ts

```diff
@@ -13,10 +16,21 @@ export function getAbsoluteUrl(href: string | undefined, parentUrl: string): str
   return new URL(href ?? "", parentUrl).toString();
 }
 
+export interface ResolvableLinkHref {
```

Comment:
```suggestion
export interface UriReferenceLookup {
```

Is it better for the various components to _extend_ this interface? Or to have it as a field?

```ts
export interface MyComponentProps extends UriReferenceLookup {
}
```

or

```ts
export interface MyComponentProps {
  uriReferenceLookup: UriReferenceLookup;
}
```